src/olympia/devhub/tests/test_tasks.py

```diff
@@ -437,7 +437,7 @@ def test_run_linter_use_temporary_file(self):
 
             assert tmpf.call_count == 2
             assert result['success']
-            assert result['warnings'] == 11
+            assert result['warnings'] == 24
```

Comment:
This is because of new `innerHTML` warnings that are detected in view/test/lib/qunit-1.12.0.js